scripts/e2e-installs.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# App temporary locations
+# See http://unix.stackexchange.com/a/84980
+temp_app_path=`mktemp -d 2>/dev/null || mktemp -d -t 'temp_app_path'`
+
+# Function to test a feature
+# Exits with return code if fails
+# See https://ouep.eu/shell/fonction-assert/
+function assert {
```

Comment:
impressive that this is done in bash, but I wonder if it couldn't have been done with jest and something like `execa` or `shelljs`

Reply:
Thinking back at it, you're right, Jest could be used to assert that the commands ran with `child_process.exeSync()` behave as expected. I will investigate!